src/rendering/renderengine.cpp

```diff
@@ -282,8 +282,6 @@ void RenderEngine::updateScene() {
         Time::ref().timeJumped(),
         _performanceManager != nullptr
     });
-
-    _scene->evaluate(_camera);
```

Comment:
Nope, it was only used for an old version of frustum culling that was commented out anyways.